.github/workflows/integration-tests.yaml

```
@@ -0,0 +1,41 @@
+on:
+  pull_request:
+    branches:
+      - dev  # Adjust to your default branch (e.g., 'master', 'develop')
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest  # Use ubuntu-latest; switch to windows-latest if needed
+
+    steps:
+    # Checkout the repository code
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)